.github/workflows/ci.yml

```diff
@@ -2,23 +2,22 @@ name: pyelftools-tests
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
Please sync this PR with the `main` branch, this should already be updated

Reply:
Done.